README.md

```diff
@@ -1,53 +1,43 @@
 # Sveltekit | Celo Composer
 
-This is a starter kit for Sveltekit. It's a perfect starter kit to get your project started on Celo blockchain. It uses TailwindCSS for styling and wagmi for onchain communication.
```

Comment:
Please still keep this information as well. Apologies if that was not clear. Just wanted to add that disclaimer to the top.